source/js/foundation/template-js-handler/article-hero-video-handler.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Adding play/pause functionality to the optional Article Page hero video section.
+ */
+
+ const articleHeroVideoHandler = () => {
```

Comment:
This file looks for a video + the play and pause buttons. If it finds them, render the pause button and play the video automatically. 

We also add a click handler on the entire video to pause it.

*Note*: At the moment, a bug with the original article pages "sticky nav" is preventing the pause/play button to be clicked. However after some discussion with Simon, we think it would be best to work together to fix this bug since he is implementing a new, updated sticky nav.